### PR TITLE
Improve usability of release script.

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

File: release-scripts/do_release.py [hunk 1]
```diff
@@ -16,7 +16,14 @@
 """do_release.py
 
 Usage:
-  do_release.py [CALICO_PLATFORM_VERSION] [CALICO_DOCKER_VERSION FELIX_VERSION LIBCALICO_VERSION LIBNETWORK_VERSION] [--dry-run]
+  do_release.py [--dry-run]
+  [--calico=<CALICO_PLATFORM_VERSION>]
```
Comment (caseydavenport) [inline release-scripts/do_release.py]: No need for "platform"

This is just the CALICO_VERSION.

File: release-scripts/do_release.py [hunk 2]
```diff
@@ -127,47 +120,59 @@ def start_release():
...
             utils.get_github_library_version("calico-containers", "https://github.com/projectcalico/calico-containers")
+
+    felix_version = arguments["--felix"]
+    if not felix_version:
         felix_version = \
             utils.get_github_library_version("felix", "https://github.com/projectcalico/felix")
-        libcalico_version = \
-            utils.get_github_library_version("libcalico", "https://github.com/projectcalico/libcalico")
+
+    # libcalico_version = arguments["--libcalico-go"]
```
Comment (caseydavenport) [inline release-scripts/do_release.py]: What's this commented stuff?

**Comment by caseydavenport** [comment]
@djosborne Looks fine - a couple comments.

I'd replace "calico platform version" with "calico version" - that number is the calico version.

**Comment by djosborne** [comment]
@caseydavenport done

**Pull request merged**